### PR TITLE
Add Colt Anaconda

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "New revolver" 
#### Purpose of change
Colt Anaconda is the best revolver in the world, can't be better, full stop. We need it in the game
#### Describe the solution
Add Colt Anaconda in three variations, for 8 inch barrel, 6 inch, and 4 inch
#### Testing
Spawned, shooted a bit, looked around to find one
#### Additional context
Data is not historical
image
image

### CODE REVIEW

File: data/json/items/gun/44.json [hunk 1]
```diff
@@ -177,5 +177,100 @@
...
+      }
+    ],
+    "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "colt_anaconda_6",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 revolver" },
+    "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has 6 inch barrel.",
```
Comment (DoctorBoomstick) [inline data/json/items/gun/44.json:235]:
```suggestion
    "description": "A large-frame revolver with a modern plastic handle.  Too large for law enforcement, it was mostly used by sport shooters and hunters, for the manufacturer deemed it to be the most accurate .44-calibre revolver in their line.  This one has a 6-inch barrel.",
```

File: data/json/items/gun/44.json [hunk 2]
```diff
@@ -177,5 +177,100 @@
...
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "colt_anaconda",
+        "name": { "str": "Colt Anaconda" },
+        "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has 6 inch barrel."
```
Comment (DoctorBoomstick) [inline data/json/items/gun/44.json:241]:
```suggestion
        "description": "A large-frame double-action revolver with a modern plastic handle.  Too large for law enforcement, it was mostly used by sport shooters and hunters, for Colt deemed it to be the most accurate .44-calibre revolver in their line.  This one has a 6-inch barrel."
```

File: data/json/items/gun/44.json [hunk 3]
```diff
@@ -177,5 +177,100 @@
...
+    "barrel_length": "153 mm",
+    "price": "1499 USD",
+    "symbol": "(",
+    "color": "dark_gray"
+  },
+  {
+    "id": "colt_anaconda_4",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 long barrel revolver" },
```
Comment (DoctorBoomstick) [inline data/json/items/gun/44.json:257]: cough cough.

File: data/json/items/gun/44.json [hunk 4]
```diff
@@ -177,5 +177,100 @@
...
+    "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has short 4 inch barrel.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "colt_anaconda",
+        "name": { "str": "Colt Anaconda" },
+        "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has short 4 inch barrel."
```
Comment (DoctorBoomstick) [inline data/json/items/gun/44.json:264]:
```suggestion
              "description": "A large-frame double-action revolver with a modern plastic handle.  Too large for law enforcement, it was mostly used by sport shooters and hunters, for Colt deemed it to be the most accurate .44-calibre revolver in their line.  This one has a shortened 4-inch barrel."
```

File: data/json/items/gun/44.json [hunk 5]
```diff
@@ -177,5 +177,100 @@
...
+    "price": "1499 USD",
+    "symbol": "(",
+    "color": "dark_gray"
+  },
+  {
+    "id": "colt_anaconda_4",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 long barrel revolver" },
+    "description": "A large frame double-action revolver with a modern plastic handle.  Too large for the law enforcement, it is mostly used by sport enthusiasts shooters and hunters, for manufacturer deemed it to be the most accurate .44 revolver in their line.  This one has short 4 inch barrel.",
```
Comment (DoctorBoomstick) [inline data/json/items/gun/44.json:258]:
```suggestion
    "description": "A large-frame revolver with a modern plastic handle.  Too large for law enforcement, it was mostly used by sport shooters and hunters, for the manufacturer deemed it to be the most accurate .44-calibre revolver in their line.  This one has a shortened 4-inch barrel.",
```

**Review by DoctorBoomstick — CHANGES_REQUESTED** [review]
Excellent to have a fresh .44. This does mean I have to stop playing until this gets merged so it’ll spawn into my current gun shops, but I'm pleased to see it. The only thing concerning here is the naming convention.
Firstly, there’s no way to differentiate the different revolver models by their branded names. You could look at the stats and, dare I say this heresy, read the description, but I think it would be convenient to have a recognisable difference in the name as well.
As for the non-branded variant, we already have an item dubbed the .44 Rivolver. Fair dues if we want to start giving functionally similar guns the same name, but I would propose a name change to something along the lines of: extended .44 Colt revolver, .44 Colt revolver, and shortened/snub-nosed .44 Colt revolver. .44 Colt revolver could be replaced with .44 Magnum, .44 hunting revolver, .44 target revolver.

File: data/json/items/gun/44.json (hunk 3 above)
Comment (GuardianDll) [reply data/json/items/gun/44.json:257]: there is 2 inch version, so technically it is not entirely false
```suggestion
    "name": { "str": ".44 short barrel revolver" },
```

**Comment by GuardianDll** [comment]
Hunting looks good for me, i agree

File: data/json/items/gun/44.json [hunk 6]
```diff
@@ -177,5 +177,100 @@
       }
     ],
     "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "colt_anaconda_8",
+    "copy-from": "pistol_revolver",
+    "looks_like": "sw_619",
+    "type": "GUN",
+    "name": { "str": ".44 long barrel revolver" },
```
Comment (GuardianDll) [inline data/json/items/gun/44.json:186]:
```suggestion
    "name": { "str": ".44 long barrel hunting revolver" },
```

File: data/json/items/gun/44.json [hunk 7]
```diff
@@ -177,5 +177,100 @@
...
+        "allowed_speedloaders": [ "44_speedloader6" ]
+      }
+    ],
+    "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "colt_anaconda_6",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 revolver" },
```
Comment (GuardianDll) [inline data/json/items/gun/44.json:234]:
```suggestion
    "name": { "str": ".44 hunting revolver" },
```

File: data/json/items/gun/44.json [hunk 8]
```diff
@@ -177,5 +177,100 @@
...
+    "barrel_length": "153 mm",
+    "price": "1499 USD",
+    "symbol": "(",
+    "color": "dark_gray"
+  },
+  {
+    "id": "colt_anaconda_4",
+    "copy-from": "colt_anaconda_8",
+    "type": "GUN",
+    "name": { "str": ".44 short barrel revolver" },
```
Comment (GuardianDll) [inline data/json/items/gun/44.json:257]:
```suggestion
    "name": { "str": ".44 short barrel hunting revolver" },
```

File: data/json/items/gun/44.json [hunk 9]
```diff
@@ -177,5 +177,100 @@
...
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE",
+        "rigid": true,
+        "ammo_restriction": { "44": 6 },
+        "allowed_speedloaders": [ "44_speedloader6" ]
+      }
+    ],
+    "melee_damage": { "bash": 10 }
```
Comment (GuardianDll) [inline data/json/items/gun/44.json:228]:
```suggestion
    "melee_damage": { "bash": 7 }
```

**Pull request merged**